{
  "schema_version": "1.4.0",
  "id": "GHSA-48pm-mhwh-g6mr",
  "modified": "2022-04-22T00:01:01Z",
  "published": "2022-04-15T00:00:43Z",
  "aliases": [
    "CVE-2022-27448"
  ],
  "details": "There is an Assertion failure in MariaDB Server v10.9 and below via 'node->pcur->rel_pos == BTR_PCUR_ON' at /row/row0mysql.cc.",
  "severity": [
    {
      "type": "CVSS_V3",
      "score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
    }
  ],
  "affected": [],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2022-27448"
    },
    {
      "type": "WEB",
      "url": "https://jira.mariadb.org/browse/MDEV-28095"
    },
    {
      "type": "WEB",
      "url": "https://lists.debian.org/debian-lts-announce/2022/09/msg00023.html"
    },
    {
      "type": "WEB",
      "url": "https://security.netapp.com/advisory/ntap-20220526-0006"
    }
  ],
  "database_specific": {
    "cwe_ids": [
      "CWE-617"
    ],
    "severity": "HIGH",
    "github_reviewed": false,
    "github_reviewed_at": null,
    "nvd_published_at": "2022-04-14T13:15:00Z"
  }
}